src/components/common/Row.svelte

```diff
@@ -35,6 +35,7 @@
             onclick={() =>
               clickData ? button.onClick(clickData) : button.onClick(rowObject)}
             disabled={button.disabled}
+            class="plausible-event-name=Button+row+object+click"
```

Comment:
I wouldn't add an event here, this is not specific enough

Reply:
removed